src/discord_app.py

```diff
@@ -19,7 +19,6 @@
 
 MAX_CHARS = 60000 # Approx 15k tokens (using 4 chars/token heuristic)
 
-init_wandb()
```

Comment:
Having the init here complicates testing because it runs on import. I believe moving it under `if __name__ == "__main__":` works